src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/convolution_backprop_data.cpp

```diff
@@ -0,0 +1,535 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/convolution_backprop_data.hpp"
+#include "shared_test_classes/single_op/convolution_backprop_data.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/filter_cpu_info.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+
+namespace {
+
+/* ============= Deconvolution params (3D) ============= */
+const std::vector<InferenceEngine::SizeVector> kernels3d = { {3, 3, 3}, {1, 1, 1} };
+const std::vector<InferenceEngine::SizeVector> strides3d = { {1, 1, 1}, {2, 2, 2} };
+const std::vector<std::vector<ptrdiff_t>> padBegins3d = { {0, 0, 0} };
+const std::vector<std::vector<ptrdiff_t>> padEnds3d = { {0, 0, 0} };
+const std::vector<InferenceEngine::SizeVector> dilations3d = { {1, 1, 1} };
```

Comment:
don't use `InferenceEngine` and `ngraph` namespaces/headers/utilities etc. It will be dropped in 2024.0

Reply:
@vurusovs corrected